docs/tutorial.md

```diff
@@ -394,7 +394,87 @@ What's going on here? Let's step through the code.
 
 (sec_extracting_individuals)=
 
-## Extracting particular SLiM individuals
+## Extracting SLiM individuals
+
+### Extracting a sample of individuals
+
+Often we would like to sample a number of individuals from a population.
+This section provides an example workflow and draws
+attention to some common errors.
+
+The first, most common method to extract individuals is simply to get all
+those that were alive at a particular time,
+using {meth}`.SlimTreeSequence.individuals_alive_at`. For instance, to get
+the list of individual IDs of all those alive at the end of the
+simulation (i.e., zero time units ago), we could do:
+
+```{code-cell}
+orig_ts = pyslim.load("example_sim.trees")
+alive = orig_ts.individuals_alive_at(0)
+
+print(f"There are {len(alive)} individuals alive in the final generation.")
+```
+
+These are individual IDs. One way to take a sample of living individuals
+and write their SNPs to a VCF is:
+
+```{code-cell}
+np.random.seed(1)
+keep_indivs = np.random.choice(alive, 100, replace=False)
+ts = pyslim.SlimTreeSequence(msprime.mutate(orig_ts, rate=1e-8, random_seed=1))
+with open("example_snps.vcf", "w") as vcffile:
+    ts.write_vcf(vcffile, individuals=keep_indivs)
+```
+
+
+### Extracting individuals after simplification
+
+In cases where we want to obtain genotypes *after* simplification, we
+need to pay attention to specific node IDs. As
```

Comment:
Isn't what we need the node IDs for the simplification step? Getting the genotypes of a subset we can do without simplification, and don't need the node IDs.

Reply:
Sorry I might not understand correctly. My thought process was, we can get genotypes for a subset without simplification and don't need the node IDs- that example is under "Extracting a sample of individuals" in my doc. But in cases where simplification _is_ used, then we risk extracting genotypes for isolated nodes, or individuals that weren't part of our intended sample (the node list given to simplify()). Am I getting this wrong?

What if I change the highlighted bit to:
        "If we simplify, we need to pay attention to specific node IDs. As mentioned earlier, SlimTreeSequence.simplify() takes a list of nodes as input:"

Reply:
Ah ha - I see what you mean. Yes, sounds good.